doc/man1/ca.pod

```diff
@@ -30,10 +30,11 @@ B<openssl> B<ca>
 [B<-md arg>]
 [B<-policy arg>]
 [B<-keyfile arg>]
-[B<-keyform PEM|DER>]
+[B<-keyform PEM|DER|ENGINE>]
```

Comment:
I would very much prefer something like an option like -pkcs12 <file>,
that replaces all key and certificate options (is usable as an alternative).
Do you agree?

Reply:
Good idea to have such an additional way for specifiying credential input in PKCS#12 format.
I've just added `-pkcs12in <file>` with 'pkcs12in' rather than 'pkcs12' to prevent potential (future) ambiguities, 
which implies `-keyfile <file> -cert <file> -keyform P12 -certform P12`.

I also updated `ca.pod` according to both above comments.
Doing so, I rearranged some of the explanations there for consistency and improved readability.